### PR TITLE
Revised timeout documentation.

### PR DESCRIPTION
**Pull request by c-bebop**
Including a description of the signaling behaviour of simex after timing out.

### CODE REVIEW

**Review by fabratu — APPROVED** [review]

**Comment by fabratu** [comment]
LGTM - thanks for the PR :)

File: docs/experiments.rst [hunk 1]
```diff
@@ -280,6 +278,11 @@ key. For example we can set the timer for an experiment to two hours as follows:
        ...
        timeout: 7200
 
+After the experiment has reached the limit of the specified timeout, the signal
+``SIGXCPU`` is send to the running process. ``SIGXCPU`` can be handled by the
```
Comment (duylethanh) [inline docs/experiments.rst:282]: little typo, should be "is sent". Same for the next sentence

**Review by duylethanh — CHANGES_REQUESTED** [review]

**Comment by fabratu** [comment]
Minor: better merge both commits to one. Fixing a typo doesn't really add anything to your previous commit.

**Comment by c-bebop** [comment]
> Minor: better merge both commits to one. Fixing a typo doesn't really add anything to your previous commit.

Done.

**Review by duylethanh — APPROVED** [review]

**Pull request merged**